stylesheets/Blind.scss

```
@@ -422,10 +425,42 @@ button.close:active:hover {
   color: #b3000f;
 }
 
-/* Reduce size for XS screens */
-@media (max-width: 375px) {
-  #thank-you-text,
-  #workflow-background-wrapper {
-    width: 85% !important;
+/* Reduce size of upload area on small screens */
+@media (max-width: 400px) {
+  .formFieldWrapper .k-upload,
+  .form-element-wrapper .k-upload {
+    width: auto !important;
   }
 }
+
+/* Reduce image size on small sceens */
+img {
+  max-width: 100% !important;
+  height: auto !important;
+}
+
+/* Reduce width of every form field on small screens */
+input[type="text"],
+input[type="password"],
+input[type="date"],
+input[type="datetime"],
+input[type="datetime-local"],
+input[type="month"],
+input[type="week"],
+input[type="email"],
+input[type="number"],
+input[type="search"]:not(.select2-search__field),
+input[type="tel"],
+input[type="time"],
+input[type="url"],
```

```suggestion
input,
```
suggestion (non-blocking): You might be able to simplify this by just specifying all inputs. (Or maybe you've already tried this and it doesn't work with SG's code for some reason…)

---

Good idea. I was able to simplify this further using only input and the dropdown selector ✅